### PR TITLE
Adding checks to travis and Dockerfile

### PR DESCRIPTION
**Pull request by R4il**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,8 +10,10 @@ env:
...
   - secure: Ieeq2tUqFMV2zmzJ1h1TXuoOTT9kenSZWMy08HfpKAXN5AC0LFI6xWpdqTf1MNxCOY+uRzLE8gGEa2BaiqGfBNqfz3I1lWNkYaIT8VyxrrRuIX6O82FTXy+KHmREy9jJ7DorgfcME3Wmrsfr1Ez5ZPrYJtggZbRJb9eJAffjRgns0ChOlckwp5qGtP2arID2cpSIPPvix8ePrsbO7468cTVy8qZQyJ3szNpgLHSKk5qkbfaaZ/8UIrTGu/BC97CjGZfD1WstYsGUgKMOxw7p6NCCt82C37RHVwbHgF7oTSl5gExmNK7yUhPgqdEYSIFoRmL3MSQ/HLukulnENXWnCGptGog9MIWQ20D0MFF+mUTEvjPbnNI8ec8JDgm8AugeSceXGLH5ODg5Kr5seaBCAm5soykoAjwU7VHNbSgOAj4MulDHcMmyvjxluYzlBweyVekN9KaNH8B1tUmibePii5AdyDsqtF7exGb1aNxSiA9QXtjDQ2kSwsgAM3+4AjlGcNjTKP3Bd41US12Il9lbHXp9B6vZniPA18M8PKBnCTiqMcQBSdl2uKnOoI5/kdBz665Ih650ooWt3n/dFvDHckdvbrm10+L/KMl0EkTToA+NwXunzp4OCmfYYLLKFRRBV++KWV2ji5y56XzP7db2dlw/IbnV7gaIlnYiri3/gds=
 after_success:
+  - ./gradlew assemble
+  - docker -v
   - docker login -u=$DOCKER_USERNAME -p=$DOCKER_PASSWORD
   - set +e
   - export REPO=aista/mapping-service
-  - docker build -f Dockerfile -t $REPO:$TRAVIS_TAG --rm=false .
+  - docker build -t $REPO:$TRAVIS_TAG -t $REPO:latest .
```
Comment (R4il) [inline .travis.yml:20]: This allows to build the image with both current tag and the "latest" default one. This way when a new release is done the latest is also updated. If there is no latest it seems that the specific tag needs to be specified.

**Comment by R4il** [comment]
[Dockerhub. Create repos and upload images from travis.](https://trello.com/c/r3Nt2pqk/84-dockerhub-create-repos-and-upload-images-from-travis)

**Comment by R4il** [comment]
[Dockerhub. Create repos and upload images from travis.](https://trello.com/c/r3Nt2pqk/84-dockerhub-create-repos-and-upload-images-from-travis)

**Comment by R4il** [comment]
[Dockerhub. Create repos and upload images from travis.](https://trello.com/c/r3Nt2pqk/84-dockerhub-create-repos-and-upload-images-from-travis)

**Review by Castyr — APPROVED** [review]

**Pull request merged**